express/app.js

```diff
@@ -29,6 +29,8 @@ app.use(require('node-sass-middleware')({
   sourceMap: true,
 }));
 app.use(express.static(path.join(__dirname, 'public')));
+// Make React Resources available
+app.use('/react', express.static(path.join(__dirname, '/../react/dist')));
```

Comment:
Ping #15 - This directory is not copied to docker. Either webpack should but the build in express/react or the folder needs to be copied.